code/datums/martial.dm

```diff
@@ -319,6 +319,7 @@
 
 /datum/martial_art/the_sleeping_carp/proc/wristWrench(mob/living/carbon/human/A, mob/living/carbon/human/D)
 	if(!D.stat && !D.stunned && !D.weakened)
+		A.do_attack_animation(D)
```

Comment:
Is there not a way to just have this happen anytime a combo is pulled off instead of pasting it in each combo?